# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchService.h
# Commit: 7f9255188d00
# Full Hash: 7f9255188d00c8f0cb4338d4604602111378de59
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-27 09:40:54
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Getting PerformanceTimingData from channel for FetchService. r=dom-worker-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138249
# ==============================================================================

diff -r 3eecad934fc1 -r 7f9255188d00 dom/fetch/FetchService.h
--- a/dom/fetch/FetchService.h	Sat Feb 26 23:24:47 2022 +0000
+++ b/dom/fetch/FetchService.h	Sun Feb 27 00:26:21 2022 +0000
@@ -102,6 +102,7 @@
     nsCOMPtr<nsICookieJarSettings> mCookieJarSettings;
     RefPtr<PerformanceStorage> mPerformanceStorage;
     RefPtr<FetchDriver> mFetchDriver;
+    SafeRefPtr<InternalResponse> mResponse;
 
     MozPromiseHolder<FetchServiceResponsePromise> mResponsePromiseHolder;
   };
